app/main.py

```diff
@@ -1 +1,45 @@
-#  write your code here
+from typing import List
+
+
+class Animal:
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        return print(f"Hello, I'm {self.name}")
```

Comment:
The `print_name` method should not return the result of the `print` function. Consider removing the `return` statement and just using `print(f"Hello, I'm {self.name}")`.